server/pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>auth</artifactId>
             <version>2.29.52</version>
         </dependency>
+        <dependency>
```

Comment:
can you put the depends version into the parent pom.xml dependency Management section? This applies to the all with version inside this pom.xml